### PR TITLE
[WIP] commonjs build task for polyfills

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: tasks/buildsources.js [hunk 1]
```diff
@@ -16,8 +30,8 @@ module.exports = function(grunt) {
 	var babel = require("babel-core");
 	var mkdir = require('mkdirp').sync;
 
-	grunt.registerTask('buildsources', 'Build polyfill sources', function() {
-
+	grunt.registerTask('buildsources', 'Build polyfill sources', function(done) {
+		var done = this.async();
```
Comment (maxnordlund) [inline tasks/buildsources.js:34]: Why shadow the `done` parameter here? Maybe just rename the parameter to `_done` instead. I also think this causes deoptimization, but is probably negligible in this case.

Comment (JakeChampion) [inline tasks/buildsources.js:34]: The shadowing was by accident, the code has since been updated :-)

Comment (maxnordlund) [inline tasks/buildsources.js:34]: No problem, always happy to help.

**Comment by JakeChampion** [comment]
I ran `npm test` locally and it passed.

**Comment by JakeChampion** [comment]
Superseded by https://github.com/Financial-Times/polyfill-service/pull/585

**Pull request closed without merging**